README.md

```diff
@@ -1,43 +1,113 @@
-# tmc-cli
+
 [![Build Status](https://travis-ci.org/tmc-cli/tmc-cli.svg?branch=master)](https://travis-ci.org/tmc-cli/tmc-cli)
 [![GitHub release](https://img.shields.io/badge/release-sprint0-brightgreen.svg?style=flat)](https://github.com/tmc-cli/tmc-cli/releases/latest)
 [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/tmc-cli/tmc-cli/master/LICENSE)
 [![Coverage Status](https://coveralls.io/repos/github/tmc-cli/tmc-cli/badge.svg?branch=master)](https://coveralls.io/github/tmc-cli/tmc-cli?branch=master)
 [![Build status](https://ci.appveyor.com/api/projects/status/np29sxc72y2f7d57?svg=true)](https://ci.appveyor.com/project/mikkomaa/tmc-cli)
+# tmc-cli
+
+TMC-CLI is the command-line client for University of Helsinki's Test My Code -framework. Test My Code is used by various online programming courses for exercise testing and submission.
+
+##Requirements
+
+* Java Runtime Environment 7
+* Linux, Mac OS X or Microsoft Windows
+  * Currently only limited support for Windows
+  * Other Unix-like systems may work, but are not tested
+* Bash is required for auto-completion - may work on other shells
```

Comment:
The above line could be changed to something like this:
If you want to use auto-completion then you have to use Bash or bash-compatible shell.